### ISSUE TITLE
Pull Request Approval broken

### ISSUE THREAD

**Issue by SchulteMarkus** [tinglesoftware/dependabot-azure-devops#1198]
``` 
# tinglesoftware/dependabot version 1.29
2024-07-09T07:37:42.9905144Z Digest: sha256:71f616dc55cb1cf35148f1c6f16439a07f5ae95132e4ea5edf7ff2ef94de4783
2024-07-09T07:37:42.9917153Z Status: Downloaded newer image for ghcr.io/tinglesoftware/dependabot-updater-npm:1.29
```

```yml
# Dependabot execution
  - task: dependabot@1
    inputs:
      autoApprove: true
      mergeStrategy: rebase
      setAutoComplete: true
```

This configuration does not work any more.
Creating a PR for a library update works fine, but when it comes to approving the PR, I get the following error:

```
2024-07-09T07:38:16.4538011Z Created pull request for @typescript-eslint/parser (#38932).
2024-07-09T07:38:16.4538154Z Auto Approving PR 38932
2024-07-09T07:38:16.4577084Z /home/dependabot/dependabot-updater/lib/tinglesoftware/dependabot/clients/azure.rb:86:in `get_with_token': uninitialized constant TingleSoftware::Dependabot::Clients::Azure::SharedHelpers (NameError)
2024-07-09T07:38:16.4577383Z 
2024-07-09T07:38:16.4577524Z             **SharedHelpers.excon_defaults(
2024-07-09T07:38:16.4577650Z               ^^^^^^^^^^^^^
2024-07-09T07:38:16.4577910Z 	from /home/dependabot/dependabot-updater/lib/tinglesoftware/dependabot/clients/azure.rb:57:in `get_user_id'
2024-07-09T07:38:16.4578258Z 	from /home/dependabot/dependabot-updater/lib/tinglesoftware/dependabot/clients/azure.rb:65:in `pull_request_approve'
2024-07-09T07:38:16.4578474Z 	from bin/update_script.rb:871:in `block in <main>'
2024-07-09T07:38:16.4578623Z 	from bin/update_script.rb:551:in `each'
2024-07-09T07:38:16.4578778Z 	from bin/update_script.rb:551:in `<main>'
```

As tinglesoftware/Dependabot worked fine until last week, I assume there is a regression, probably in v1.29.5 https://github.com/tinglesoftware/dependabot-azure-devops/releases/tag/1.29.5 --> https://github.com/tinglesoftware/dependabot-azure-devops/compare/1.29.4...1.29.5

**Comment by rhyskoedijk** [comment]
My fault sorry, https://github.com/tinglesoftware/dependabot-azure-devops/pull/1199 will fix it.

**Comment by SchulteMarkus** [comment]
> My fault sorry, #1199 will fix it.

No worries, mate, you're doing a fine job here.

**Comment by rhyskoedijk** [comment]
@SchulteMarkus fixed in 1.29.6
